cmd/adhoc/cmd/download.go

```diff
@@ -31,16 +50,40 @@ func init() {
 		0,
 		"Download data from the last X years",
 	)
-
 	root.AddCommand(download)
 }
 
 func onRun(cmd *cobra.Command, args []string) error {
+	Month, _ := cmd.Flags().GetInt(cli.Months)
+	Year, _ := cmd.Flags().GetInt(cli.Years)
+	cliDay, err := cmd.Flags().GetInt(cli.Days)
+
+	if cliDay < 2 {
+		logrus.Error("No data available.")
+		logrus.Error("Try 3 days or greater.")
```

Comment:
I have my doubts when it comes to logging, but I think it is best to join all messages related to an error into a single one. Otherwise it looks like you have more problems than you do.
`Error: No data available. Try again with...`
vs
```
Error: No data available.
Error: Try again with...
```

Reply:
Very good point, fixed.